packages/PreviewCard/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@tds/community-preview-card",
+  "version": "1.0.0-preview",
+  "description": "",
+  "main": "index.cjs.js",
+  "module": "index.es.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/telus/tds-community.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "TELUS digital",
+  "engines": {
+    "node": ">=8"
+  },
+  "bugs": {
+    "url": "https://github.com/telus/tds-community/issues"
+  },
+  "homepage": "http://tds.telus.com",
+  "peerDependencies": {
+    "react": ">=15",
+    "react-dom": ">=15",
+    "styled-components": "^4.2.0"
+  },
+  "dependencies": {
+    "@tds/core-colours": "^1.1.1",
+    "jest-styled-components": "^6.3.1",
```

Comment:
`jest-styled-components` should be a devDependency in the root [package.json](https://github.com/telus/tds-community/blob/master/package.json) just like in [core](https://github.com/telus/tds-core/blob/master/package.json#L114)